app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
+end
```

Comment:
- [ ] The `belong_to` should belong to the author and the author is associated with the User table cause in the table you have `author_id` not `user_id`, it can be similar to 👉🏾` belongs_to :author, class_name: 'User'`. Kindly fix that 😁
- [OTIPNAL] Same remark about using the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) for the `update_comments_counter`.
The code can be similar to 👇 

```ruby
  after_save :update_comments_counter

  private

  def update_comments_counter
    author.increment!(:comments_counter)
  end
```